egs/swbd/s5c/local/chain/run_tdnn_6z.sh

```diff
@@ -55,7 +55,7 @@ final_layer_normalize_target=0.5
 num_jobs_initial=3
 num_jobs_final=16
 minibatch_size=128
-relu_dim=576
+relu_dim=1024
```

Comment:
OK, keep all hyper-parameters the same as in 6z. Just change splice-indexes
and relu-dim.

On Wed, Mar 23, 2016 at 11:55 PM, Daniel Galvez <notifications@github.com>
wrote:

> In egs/swbd/s5c/local/chain/run_tdnn_6z.sh
> <https://github.com/kaldi-asr/kaldi/pull/612#discussion_r57272162>:
>
> > @@ -55,7 +55,7 @@ final_layer_normalize_target=0.5
> >  num_jobs_initial=3
> >  num_jobs_final=16
> >  minibatch_size=128
> > -relu_dim=576
> > +relu_dim=1024
>
> Yep, got it. I won't run the other experiment you proposed, as I get your
> point now. Sorry for misunderstanding.
>
> I'm going to keep num_epochs=4 and learning_rate the same as in 6z. The
> cross entropy model has a higher learning rate, but I think it makes sense
> to make smaller steps for a discriminative model.
>
> Will get the experiment going now. Will reopen this if I see problems.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/612/files/f813903fd09e194096123d2c588662dc66c0e4e6#r57272162>
>